pynamodb/models.py

```diff
@@ -257,7 +257,8 @@ def get_hash_key(self):
 
     def get_range_key(self):
         key_name = self._range_keyname
-        if key_name is not None and hasattr(self, key_name):
+        if key_name is not None:
+            return getattr(self, key_name, None)
             return getattr(self, key_name)
```

Comment:
of course remove this line :) github doesn't let you make suggestions that replace multiple lines

Reply:
heh